pkg/kube/controllers/extendedjob/job_creator.go

```diff
@@ -103,13 +103,19 @@ func (j jobCreatorImpl) Create(ctx context.Context, eJob ejv1.ExtendedJob, podNa
 	if err != nil {
 		return false, errors.Wrapf(err, "could not generate job name for eJob '%s'", eJob.Name)
 	}
+
+	backoffLimit := int32(2)
```

Comment:
Oh, I thought we decided to not change the defaults during the call? Or did I miss anything?

Reply:
Added in the story -  https://www.pivotaltracker.com/n/projects/2192232/stories/167866215